libs/features/cdt/cdt.css

```diff
@@ -14,7 +14,7 @@
 }
 
 .timer-label {
-  font-size: var(--type-body-s-size);
+  font-size: 16px;
```

Comment:
The variables should likely not be replaced, what was the intention here? Same comment for other variable replacements in this file.

At one point, we'll update all the variables to follow the Spectrum 2 guidelines, we want to make sure we avoid additional work at that point by having values already mapped to the ones in `styles.css`

Reply:
reverted